src/test/groovy/org/prebid/server/functional/service/PrebidServerService.groovy

```diff
@@ -320,4 +329,10 @@ class PrebidServerService {
         }
         filteredLogs
     }
+
+    private static RequestSpecification buildAndGetRequestSpecification(String uri, BasicAuthScheme authScheme) {
```

Comment:
If you want to make method universal, use the `AuthentificationScheme` instead of `BasicAuthScheme`